octobot_trading/exchanges/config/exchange_config_data.py

```diff
@@ -82,6 +84,50 @@ def set_historical_settings(self):
     def get_shortest_time_frame(self):
         return self.traded_time_frames[-1]
 
+    def initialize_exchange_event_tree(self):
+        tree_provider = commons_tree.EventProvider.instance()
+        for topic in trading_constants.DEFAULT_FUTURES_INITIALIZATION_EVENT_TOPICS \
+                if self.exchange_manager.is_future \
+                else trading_constants.DEFAULT_INITIALIZATION_EVENT_TOPICS:
+            if topic in (
+                    commons_enums.InitializationEventExchangeTopics.POSITIONS,
```

Comment:
I don't think we should create position and contract related paths in spot as we don't wait paths that do not exist.

Reply:
Right, that's why they are not in `DEFAULT_INITIALIZATION_EVENT_TOPICS`
![image](https://user-images.githubusercontent.com/9078616/191720481-4d4b3479-dfae-4bff-9074-88e30283079a.png)

Reply:
Right ! My bad